app-crypt/certbot/certbot-9999.ebuild

```diff
@@ -146,40 +146,73 @@ RDEPEND="
 # 	)
 # "
 
+# Note: "docs" is not an actual directory under "S", they are actually
+# under each modules, see python_compile_all redefinition, but keep
+# this instruction enabled for dependency configuration.
+distutils_enable_sphinx docs \
+	dev-python/sphinx-rtd-theme
 distutils_enable_tests pytest
 
-certbot_dirs=()
+CERTBOT_DIRS=()
+# Stores temporary modules docs in each subdirectories,
+# will be used for HTML_DOCS
+CERTBOT_DOCS="${T}/docs"
 
-my_certbot_dirs_listing() {
+src_prepare() {
+	default
+
+	# set CERTBOT_DIRS
 	local base module
 	for base in "${CERTBOT_BASE[@]}"; do
-		certbot_dirs+=("${base}")
+		CERTBOT_DIRS+=("${base}")
 	done
 	for module in "${CERTBOT_MODULES_EXTRA[@]}"; do
 		use "certbot-${module}" \
```

Comment:
```suggestion
		use "certbot-${module}" &&
```

Hate backslashes.

Reply:
In other languages, some people like me prefer to display operators at the beginning of the line, but I agree Bash slightly differs.